sdk/physical/inmem/cache_test.go

```diff
@@ -328,3 +328,45 @@ func TestCache_Refresh(t *testing.T) {
 		t.Fatalf("expected value baz, got %s", string(r.Value))
 	}
 }
+
+// TestCache_UpdateFromStorage fetches a nil value from cache, updates
+// the underlying storage and checks the cache value on a subsequent lookup
+func TestCache_UpdateFromStorage(t *testing.T) {
```

Comment:
is this supposed to be testing the change made to `cache.go`? if so, i'm not sure how it does so. i'd think we'd want to write something nil and then verify that it's not in the cache?

Reply:
chatted with @swayne275 , verified this test is checking the change in cache.go appropriately, closing this out.